module_1/src/App.css

```diff
@@ -0,0 +1,3 @@
+.selected{
+  background-color: purple;
```

Comment:
As you are using this class and rule only once, at the `GenreSelect`, please put the CSS next to that component and import it there.